New Terms: Nmap no ping, no port (#1300)
Codecademy/docs: content/cybersecurity/concepts/nmap/terms/no-port-scan/no-port-scan.md

---
Title: 'No Port Scan'
Description: 'A Nmap no port scan prevents Nmap from doing a port scan after host discovery.'
Subjects:
  - 'Computer Science'
  - 'Networking'
Tags:
  - 'Networking'
  - 'Cybersecurity'
  - 'Ethical Hacking'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/scan-systems-with-nmap'
---

> A Nmap **no port scan** prevents Nmap from doing a port scan after host discovery. It is specified with the `-sn` option. Used by itself it is also known as a "ping" scan. It is less intrusive and attracts less attention than a full port scan.

Reviewer: ```suggestion
A **no port scan** prevents Nmap from doing a port scan after host discovery. It is specified with the `-sn` option. Used by itself, it is also known as a "ping" scan. It is less intrusive and attracts less attention than a full port scan.
```

> **Note:** This option is often combined with a [no ping scan (`-Pn`)](https://www.codecademy.com/resources/docs/cybersecurity/nmap/no-ping-scan).

## Syntax

```pseudo
nmap -sn <target>
```

This performs a scan of `<target>` without doing a port scan.

## Example

> The following shows the output of `nmap -sn scanme.nmap.org`:

Reviewer 2: ```suggestion
The following example runs a no ping scan  on the site [scanme.nmap.org](http://scanme.nmap.org/):

```shell
nmap -sn scanme.nmap.org
```
```

> **Note:** `scanme.nmap.org` is a domain set up explicitly for people to test Nmap with.

> 

Reviewer 2: ```suggestion

This results in the following output:

```
